stdlib/3/unittest/__init__.pyi

```diff
@@ -92,51 +88,56 @@ class TestCase:
                           msg: Any = ...) -> _AssertRaisesContext[_E]: ...
     @overload
     def assertWarns(self,  # type: ignore
-                    exception: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                    expected_warning: Union[Type[Warning], Tuple[Type[Warning], ...]],
                     callable: Callable[..., Any],
                     *args: Any, **kwargs: Any) -> None: ...
     @overload
     def assertWarns(self,
-                    exception: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                    expected_warning: Union[Type[Warning], Tuple[Type[Warning], ...]],
                     msg: Any = ...) -> _AssertWarnsContext: ...
     @overload
     def assertWarnsRegex(self,  # type: ignore
-                         exception: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                         expected_warning: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                         expected_regex: Union[str, bytes, Pattern[str], Pattern[bytes]],
                          callable: Callable[..., Any],
                          *args: Any, **kwargs: Any) -> None: ...
     @overload
     def assertWarnsRegex(self,
-                         exception: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                         expected_warning: Union[Type[Warning], Tuple[Type[Warning], ...]],
+                         expected_regex: Union[str, bytes, Pattern[str], Pattern[bytes]],
                          msg: Any = ...) -> _AssertWarnsContext: ...
     def assertLogs(
         self, logger: Optional[logging.Logger] = ...,
         level: Union[int, str, None] = ...
     ) -> _AssertLogsContext: ...
     def assertAlmostEqual(self, first: float, second: float, places: int = ...,
                           msg: Any = ..., delta: float = ...) -> None: ...
+    @overload
+    def assertNotAlmostEqual(self, first: float, second: float,
+                             places: int = ..., msg: Any = ...) -> None: ...
```

Comment:
Why did you remove the `delta` argument? It seems to exist in recent Python 3.

Also, I'm not sure why the overloads are necessary.

Reply:
Already commented.